### DISCUSSION TITLE
사이드바 너비 변경

### DISCUSSION THREAD

**Discussion by JOO1999** [pronist/hELLO#166] [Q&A] [answered]
사이드바 너비가 현재 1400으로 잡혀있는것으로 알고있습니다.

이부분을 1500정도로 늘리고싶은데 늘리는 방법과 늘렸을 경우 모바일에서의 설정값들과 문제가 날 수 있을까요?

**Comment by pronist** [comment, marked as answer]
@JOO1999 안녕하세요. 가급적이면 건들지 않는 것을 추천드립니다. 이를 늘리려면 CSS 를 수정하셔야 하는데, 사이드바의 크기와 상호작용하는 요소(스킨 커버, 글 읽기 모드 등)들에 대한 수정이 필요합니다. 이는 생각보다 번거로운 일이므로 권하지 않습니다.

**Reply by JOO1999** [reply]
그렇군요.. 답변 감사합니다!!
